src/backend/utils/fetch_with_cookie.ts

```diff
@@ -0,0 +1,18 @@
+import { DEV_PORTAL_URL } from 'common/constants'
+import getPartitionCookies from './get_partition_cookies'
+
+export async function fetchWithCookie(url: string, method: string) {
+  const cookieString = await getPartitionCookies({
+    partition: 'persist:auth',
```

Comment:
Can we add the partition as the argument?

Reply:
yup. good idea